src/templates/default/reviews/review_change.html

```diff
@@ -0,0 +1,161 @@
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".reviews.stage"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input id="reviews.stage" name="reviews.stage" type="number" min="0" max="2"
+								placeholder="Int" required="required">
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.disable.after"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input name="proposals.disable.after" type="datetime-local" step="1"
+								placeholder="yyyy-mm-dd hh:mm:ss" required="required">
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						timezone
+						<div class="input-customized-size pull-right">
+							<select name="DropDownTimezone" id="DropDownTimezone">
+								<option value="-12:00">(GMT -12:00) AoE</option>
+								<option value="-11:00">(GMT -11:00) Midway Island</option>
+								<option value="-10:00">(GMT -10:00) Hawaii</option>
+								<option value="-09:00">(GMT -9:00) Alaska</option>
+								<option value="-08:00">(GMT -8:00) Pacific Time (US &amp; Canada)</option>
+								<option value="-07:00">(GMT -7:00) Mountain Time (US &amp; Canada)</option>
+								<option value="-06:00">(GMT -6:00) Central Time (US &amp; Canada)</option>
+								<option value="-05:00">(GMT -5:00) Eastern Time (US &amp; Canada)</option>
+								<option value="-04:00">(GMT -4:00) Atlantic Time (Canada)</option>
+								<option value="-03:30">(GMT -3:30) Newfoundland</option>
+								<option value="-03:00">(GMT -3:00) Brazil, Buenos Aires</option>
+								<option value="-02:00">(GMT -2:00) Mid-Atlantic</option>
+								<option value="-01:00">(GMT -1:00 hour) Azores</option>
+								<option value="+00:00">(GMT) Western Europe Time</option>
+								<option value="+01:00">(GMT +1:00 hour) Brussels</option>
+								<option value="+02:00">(GMT +2:00) South Africa</option>
+								<option value="+03:00">(GMT +3:00) Baghdad</option>
+								<option value="+03:30">(GMT +3:30) Tehran</option>
+								<option value="+04:00">(GMT +4:00) Abu Dhabi</option>
+								<option value="+04:30">(GMT +4:30) Kabul</option>
+								<option value="+05:00">(GMT +5:00) Ekaterinburg</option>
+								<option value="+05:30">(GMT +5:30) Bombay</option>
+								<option value="+05:45">(GMT +5:45) Kathmandu</option>
+								<option value="+06:00">(GMT +6:00) Almaty</option>
+								<option value="+07:00">(GMT +7:00) Bangkok</option>
+								<option value="+08:00">(GMT +8:00) Taiwan {{ default_tz }}</option>
+								<option value="+09:00">(GMT +9:00) Tokyo, Seoul</option>
+								<option value="+09:30">(GMT +9:30) Adelaide, Darwin</option>
+								<option value="+10:00">(GMT +10:00) Eastern Australia</option>
+								<option value="+11:00">(GMT +11:00) Magadan</option>
+								<option value="+12:00">(GMT +12:00) Auckland</option>
```

Comment:
Why not generating this dynamically with `pytz`? This feels awkward and inflexible.

Reply:
Thanks for you advise. I studied django docoment timezone and use it in 6ef2f5bae98217c0e625ae323515acc69f2afd10.